client/src/components/layout/TripTile.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Link } from "react-router-dom"
```

Comment:
Doesnt look like this `Link` is being used so we can remove the import statement